Expensify/App
# Add Guides call request flow to workspace card configuration screen

cc @jasperhuangg 

### Details
This PR does two things
1. Generalizes the ability to queue an inbox call, so that we can continue to use the same modal for different taskIDs
2. Adds the call button to the Company Cards page of the workspace modal

### Fixed Issues
$ https://github.com/Expensify/Expensify/issues/173720#issuecomment-897795837

### Tests
## Regression Tests
1. Navigate to your concierge DM
1. Click the call button
1. _Verify_ it opens the request a call modal
1. Fill out the form and submit it
1. _Verify_ you have a new `Expensiworks/InboxCallUser` job with the taskID `ExpensifyCashConciergeDM` in your jobs table
1. Navigate to a non-concierge DM
1. Click the call button
1. _Verify_ you are given the choice between Zoom and Google Meet

## Feature Tests
1. From the FAB, click the `New Workspace` button
1. [mobile only] Click the `Company Cards` button
1. _Verify_ you see the call button 
1. [Web & Desktop only] Hover the call button, verify you see the tooltip
1. Click the call button
1. _Verify_ it opens the inbox call dialog
1. Fill out the form and submit it
1. _Verify_ you have a new `Expensiworks/InboxCallUser` job with the taskID `WorkspaceCompanyCards` in your jobs table 

### QA Steps
1. From the FAB, click the `New Workspace` button
1. [mobile only] Click the `Company Cards` button
1. _Verify_ you see the call button
1. [Web & Desktop only] Hover the call button, verify you see the tooltip
1. Click the call button
1. _Verify_ it opens the inbox call dialog

Repeat for each platform

### Tested On


### Screenshots

#### Web
![image](https://user-images.githubusercontent.com/22447860/129934403-4b61c72e-dc6e-443f-bfb5-ca928cd4954c.png)

Also tested the translation on web
![image](https://user-images.githubusercontent.com/22447860/129934314-6a89ee13-23fd-4ccd-8261-14a68d4710ec.png)

#### Mobile Web
![image](https://user-images.githubusercontent.com/22447860/129934789-848f79cb-43fa-4935-a7f0-f3810f00953a.png)

#### Desktop
![image](https://user-images.githubusercontent.com/22447860/129935502-ef4fbd51-c9f7-43af-b316-7040831b2546.png)

#### iOS
![image](https://user-images.githubusercontent.com/22447860/129935912-0f09d2fe-5984-46d9-bf70-8fe13871605d.png)

#### Android
![image](https://user-images.githubusercontent.com/22447860/129936291-4b1479f5-a785-44cc-b67b-6c6ac44dcaab.png)
